src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -197,12 +208,18 @@ bool ResetGlobalPropertiesRequest::PrepareResponseParameters(
   } else {
     result =
         PrepareResultForMobileResponse(ui_properties_info, tts_properties_info);
+    result = (result || PrepareResultForMobileResponse(ui_properties_info,
```

Comment:
@RomanReznichenkoLuxoft I suggest you to create an overload for `PrepareResultForMobileResponse` in the base class which accepts three parameters and use this overload here in one line